packages/flutter_blue/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: flutter_blue_tizen
+description: A new flutter plugin project.
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.20.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_blue: ^0.8.0
```

Comment:
Unused dependency. Please remove.

Reply:
Could you add the following removed lines back to the pubspec?

```yaml
dependencies:
  flutter:
    sdk: flutter

dev_dependencies:
  flutter_lints: ^1.0.0
```

- flutter: Might be parsed by pub.dev.
- flutter_lints: Used by the `analysis_options.yaml` file.